Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -211,6 +169,74 @@ extension BrowserViewController {
       }
     }
   }
+  
+  private func bookmarksButton(_ menuController: MenuViewController) -> MenuItemButton {
```

Comment:
Like I mentioned in the previous comment we should have one functions which handles `MenuItemButton` creation and for helping this parametrized creation we have to create `enum` which can help us to determine the details of every button like

```
  // MARK: MenuButtonType
  
  enum MenuButtonType {
    case news
    case talk
    case wallet(subtitle: String)
    
      .
      .
      .

    var icon: UIImage {
      switch self {
        case .news:
          return UIImage(named: "menu_brave_news", in: .current, compatibleWith: nil)!.template
        case .talk:
          return UIImage(named: "menu-brave-talk", in: .current, compatibleWith: nil)!.template
        case .wallet(_):
          return UIImage(named: "menu-crypto", in: .current, compatibleWith: nil)!.template
      }
    }
    
    .
    .
    .
  }
```